source/components/GlobalNavigation/hooks/useLazyLoad.js

```diff
@@ -7,35 +7,22 @@ const hasAlmostScrolledToTheBottom = element => (
 );
 
 const getOnScrollListener = handler => (event) => {
+    /* istanbul ignore else */
     if (hasAlmostScrolledToTheBottom(event.target)) {
         handler();
     }
 };
 
-const onMouseWheel = (event) => {
```

Comment:
Do we not need this handler anymore?

Reply:
probably not.
1. `mousewheel` and `DOMMouseScroll` are non-standard => MDN suggests using `wheel` instead
2. they were used to differently (don't know why) support mouse devices via `mousewheel`, but it's already covered via `scroll` event. I've tried to reverse-engineer this code but didn't find any specific reasons to use it, so considering our current tech stack and moderns browsers I decided to remove it until we find some good reason to bring it back.